src/nbt.rs

```diff
@@ -75,3 +78,21 @@ impl Deref for Nbt {
         &self.root_tag
     }
 }
+
+impl From<NbtCompound> for Nbt {
```

Comment:
Could you add something like that but for compound, so users could eaisier modify nbt tags

Reply:
Good idea.